src/Handlers/VIewCacheHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
Small type on file name `VIewCacheHandler.php` no be renamed `ViewCacheHandler.php`